ruby/ql/lib/ruby.dbscheme

```diff
@@ -114,6 +114,10 @@ databaseMetadata(
   string value: string ref
 );
 
+overlayChangedFiles(
+  string path: string ref
```

Comment:
Why `string` and not `@file` ?

Reply:
Because this will be populated by the CLI from the list of changed files calculated by the CodeQL Action, which in turn uses `git` to diff against the base branch, and it seems quite possible that there will be paths for files that aren't in the database and therefore don't have a `@file` entity.

If we wanted to use `@file`, we'd have to consider making the CLI also emit `files` tuples to ensure db consistency. It seems simpler just to use strings, at the low cost of adding an extra join in the discard predicates.